CardBox/CardGameEngine/Actions/PlayCardAction.swift

```diff
@@ -11,23 +11,18 @@ struct PlayCardAction: Action {
     let target: GameplayTarget
 
     func executeGameEvents(gameRunner: GameRunnerReadOnly) {
```

Comment:
Is it possible to revert this file back to the original changes?

The canPlay method should be attached to the players not the cards, otherwise, we can't handle the case where multiple cards are played.

Reply:
Done!